### PR TITLE
Add scripts to create the daily builds for Mac OS.

### PR DESCRIPTION
**Pull request by camilasan**
These scripts work. 

I do not fully understand why I had to use $(security find-generic-password -w -s "NOTARIZATION_ACCOUNT") instead of just "@keychain:NOTARIZATION_PASSWORD".

### CODE REVIEW

**Comment by FlexW** [comment]
> I do not fully understand why I had to use $(security find-generic-password -w -s "NOTARIZATION_ACCOUNT") instead of just "@keychain:NOTARIZATION_PASSWORD".

I think that syntax was maybe deprecated or something like that.

File: osx/daily/watcher.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
+<?php
+
+// Time to sleep between checks
+$sleeptime = 60;
+// Command to run
+//$command = '/Users/builder/daily/build.sh';
```
Comment (FlexW) [inline osx/daily/watcher.php:6]: maybe you could remove that comment

**Review by FlexW — APPROVED** [review]

File: osx/daily/build.sh [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+export PATH=$QT_DIR/bin/:$PATH
+export OPENSSL_ROOT_DIR=$(brew --prefix openssl@1.1)
+
+mkdir install
+
+#Build QtKeychain
+mkdir qtkeychain
+cd qtkeychain
+
+cmake \
```
Comment (mgallien) [inline osx/daily/build.sh:38]: not blocking from me but could we try to use the Ninja generator ?

**Review by mgallien — APPROVED** [review]

File: osx/daily/build.sh (hunk 2 above)
Comment (FlexW) [reply osx/daily/build.sh:38]: but make sure it is installed

Comment (camilasan) [reply osx/daily/build.sh:38]: I would propose to merge it so we have it done and versioned somewhere. Later we can improve it.

**Pull request merged**